ydb/aio/credentials.py

```diff
@@ -55,8 +54,9 @@ def submit(self, callback):
             asyncio.ensure_future(self._wrapped_execution(callback))
 
 
-@six.add_metaclass(abc.ABCMeta)
-class AbstractExpiringTokenCredentials(credentials.AbstractExpiringTokenCredentials):
+class AbstractExpiringTokenCredentials(
+    credentials.AbstractExpiringTokenCredentials, abc.ABC
```

Comment:
i think there is no need to inherit from abc.ABC becouse we have already inherited from abc-class.